infrasnoop/ii.tf

```diff
@@ -50,18 +54,18 @@ data "coder_parameter" "space-image" {
   type         = "string"
 }
 
-data "coder_parameter" "infrasnoop-image" {
-  name         = "infrasnoop-image"
+data "coder_parameter" "infrasnoop_image" {
+  name         = "infrasnoop_image"
```

Comment:
I'm seeing a pattern of folks consistently using "_" instead of "-" in terraform names. It does make it easier to read when referring to them. Especially 'coder_parameters"